### PR TITLE
Add flatpak manifest

### PR DESCRIPTION
**Pull request by albfan**
Let the kids play!

### CODE REVIEW

**Comment by Alecaddd** [comment]
@bilelmoussaoui is this good to merge?

File: com.github.akiraux.interactive-goocanvas.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      "/lib/debug",
+      "/share/vala",
+      "/man",
+      "*.a",
+      "*.la"
+   ],
+   "finish-args": [
+      "--share=ipc",
+      "--socket=wayland",
+      "--share=network"
```
Comment (bilelmoussaoui) [inline com.github.akiraux.interactive-goocanvas.json]: What do we need network for here? 
Plus, does it not work with x11?

**Comment by bilelmoussaoui** [comment]
@Alecaddd I left a review. Once it's fixed, this can be merged

File: com.github.akiraux.interactive-goocanvas.json (hunk 1 above)
Comment (Alecaddd) [reply com.github.akiraux.interactive-goocanvas.json]: I don't think we need Network access now. In the future for sure when we'll implement some sort of cloud syncing feature, but right now I can't think of any scenario where we need network access.

Regarding x11 or wayland, I've no idea so I let you guys decide :sweat_smile:

Comment (albfan) [reply com.github.akiraux.interactive-goocanvas.json]: fixed

**Comment by albfan** [comment]
@bilelmoussaoui, please merge yourself if you find it correct now.

**Pull request merged**